scripts/install_third_party_libs.py

```diff
@@ -35,6 +35,7 @@
     ('six', '1.16.0', os.path.join('third_party', 'python_libs')),
     ('certifi', '2021.10.8', os.path.join(
         TOOLS_DIR, 'certifi-2021.10.8')),
+    ('typing-extensions', '4.0.1', os.path.join('third_party', 'python_libs')),
```

Comment:
can we download this through `requirement.txt`?

Reply:
No, because otherwise when _feconf.py_ is imported through some file in _scripts_ it will fail.

Reply:
Thanks, @vojtechjelinek as you said, I tried to import the module in [this commit](https://github.com/oppia/oppia/pull/14369/commits/e37dfa85a25bff03787c45411aaf09de4cc6c052) but it was not worked.